apps/webClient/src/components/View/index.tsx

```diff
@@ -59,6 +61,13 @@ const useStyles = makeStyles(theme => ({
     backgroundColor: YELLOW,
     background: YELLOW_GRADIENT,
   },
+  footer: {
+    marginTop: "auto",
```

Comment:
curious what the marginTop does for us

Reply:
Ah. The flexGrow 1 does the trick already. I was using this to push the footer down, but it appears that isn't necessary.